### ISSUE TITLE
The Healing potion dropped by the witch does not stack with the brewable version

### ISSUE THREAD

**Issue by Battosay52** [BTW-Community/BTW-Public#370]
(latest CE no mods)
Potions dropped by the witch are #0373/16341
The one you get from a brewing stand is #0373/8197
This lead to further issues like for splash variants, but just fixing this should help the other issues regarding health potions.
(and holy shit, potions are a mess -_-)

**Comment by LannyRipple** [comment]
Interesting.  Witches get handed one of their health potions when they take damage.  I'm not sure they were ever supposed to drop one.  Minecraft doesn't have a potion in the drop list and neither does BTW.

**Comment by prof-ion** [comment]
Potions are indeed a mess holy shit. I spent some time looking through item IDs. Interesting it looks like you can brew the dropped potion into the normal variant with a netherwart. Which seems SUPER weird. 
There are a LOT of duplicate potions as well. 

I can also confirm that these do drop sometimes from witches, I got one from my mobtrap randomly and was really confused as I didn't think they were meant to drop them.

**Comment by LannyRipple** [comment]
Same issue in the 1.6.4 code.  Also normal drops (including glowstone and redstone) will need a cleanup pass.

**Comment by Hirachosan** [comment]
its not a regular drop because they create the item, put it in their inventory and then drink it. killing a witch while its drinking it will drop the potion. FC knew about this and if he didn't like that he'd have removed it. Personally i dont see an issue with it either, its an interesting way to get potion drops

**Comment by prof-ion** [comment]
> its not a regular drop because they create the item, put it in their inventory and then drink it. killing a witch while its drinking it will drop the potion. FC knew about this and if he didn't like that he'd have removed it. Personally i dont see an issue with it either, its an interesting way to get potion drops

It is an interesting way to get them indeed! But the lack of stacking isn't ideal. And that was the issue that Battosay was mentioning.

**Comment by LannyRipple** [comment]
You suggested it can stack if you use a netherwart to turn it into a normal potion?  Not sure how someone would ever learn that though short of a wiki page but will be using it next time I find a witch potion.

**Comment by prof-ion** [comment]
Yeah I found it thanks to craft guide. You can convert the potion to the other item ID that way. Note that I haven't actually tested that yet. But as far as I can tell it SHOULD work. Just have to re-brew them.
